ToDo/ToDo/ViewController.swift

```diff
@@ -46,12 +49,38 @@ class ViewController: UIViewController {
         present(alert, animated: true)
     }
     
+    
+    
+    func save(item: String) {
+        
+        guard let appDelegate =
+            UIApplication.shared.delegate as? AppDelegate else {
+                return
+        }
+        
+        let managedContext =
+            appDelegate.persistentContainer.viewContext
+        
+        let entity =
+            NSEntityDescription.entity(forEntityName: "ToDoList",
+                                       in: managedContext)!
+        
+        let toDoListItem = NSManagedObject(entity: entity,
+                                           insertInto: managedContext)
```

Comment:
🍰Same here, I'd put all these on one line. (Jeff Kelly, for example, would disagree with this.) It's 2019! We have wide-screen monitors! Spread out! We have space!